sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringServiceImpl.java

```diff
@@ -250,9 +312,22 @@ public Mono<SpringService> createResourceAsync() {
             });
     }
 
+    @Override
+    public Mono<Void> afterPostRunAsync(boolean isGroupFaulted) {
+        clearCache();
+        if (isGroupFaulted) {
+            return Mono.empty();
+        }
+        return refreshAsync().then();
```

Comment:
Does refresh needed? It is an extra GET?

Reply:
Removed.